tests/test_bert_regressor_truncated.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from shutil import rmtree
+
+import torch
+
+from belt_nlp.bert_regressor_truncated import BertRegressorTruncated
+MODEL_PARAMS = {"batch_size": 1, "learning_rate": 5e-5, "epochs": 1, "device": "cpu"}
+
+
+def test_fit_and_predict():
+    """The test is quite naive, but it goes through all the methods."""
+    params = MODEL_PARAMS
+    model = BertRegressorTruncated(**params)
+    x_train = ["good", "super", "ok", "great"]
+    y_train = [1] * len(x_train)
+
+    x_test = ["nice", "cool"]
+
+    model.fit(x_train, y_train)
+    scores = model.predict(x_test)
+
+    assert scores.shape == torch.Size([2, 1])
+
+
+def test_regression_order():
```

Comment:
I think that for both classifiers and regressors `prediction_order` is the appropriate name.

`predictions` is used in the docstring anyway ;)